src/backend/marsha/urls.py

```diff
@@ -40,6 +47,8 @@
     # Admin
     path(f"{admin_site.name}/", admin_site.urls),
     # LTI
+    path("lti/select/", LTISelectView.as_view(), name="select_lti_view"),
+    path("lti/respond/", LTIRespondView.as_view(), name="respond_lti_view"),
```

Comment:
Maybe call it "response" to keep the same vocabulary as them?
```suggestion
    path("lti/response/", LTIRespondView.as_view(), name="response_lti_view"),
```

Reply:
Not sure, as it is the act of responding, not getting a response…

Reply:
:+1: